avdevice/__init__.py

```diff
@@ -36,33 +36,38 @@
 
 import serial
 import socket
+from .AVDeviceInit import Init
+
+import sys
+sys.path.append('/usr/local/lib/python3.5/dist-packages/pysrc')
+import pydevd
```

Comment:
I guess `pydevd` module was only used while developing the plugin, right? The static path added to `sys.path` too, I guess.